test/unit/spec/v1/EnrollSms_spec.js

```diff
@@ -724,7 +724,13 @@ Expect.describe('EnrollSms', function() {
         });
     });
     itp('shows error if error response on verification', function() {
-      return sendValidCodeFn()
+      return sendValidCodeFn({
+        i18n: {
+          en: {
+            'errors.E0000068': 'Invalid Passcode or Answer'
```

Comment:
Looks like the translation string and this response mismatch?

"errors.E0000068 = Invalid Passcode/Answer"
vs
'errors.E0000068': 'Invalid Passcode or Answer'

Does the test pass?

Reply:
Yes, they mismatch. 
`Invalid Passcode/Answer` is the original error message that comes from server response ([this mock](https://github.com/okta/okta-signin-widget/blob/master/test/unit/helpers/xhr/MFA_ENROLL_ACTIVATE_errorActivate.js)).
`Invalid Passcode or Answer` is the translation string, I made it different in purpose to demonstrate that the final error message should be translated.
The test passes.

Reply:
Good point Lester, it is definitely better to have a conjunction like "or" instead of a slash. 

Regarding if it is just for tests, our mock can have something like "Invalid Passscode (API error)", so we know where each comes from, as opposed to a small difference like the slash vs "or"